{"error": {"error_code": 5, "error_msg": "User authorization failed: access_token was given to another ip address", "error_text": "Failed to connect to\u00a0network. Please close the\u00a0page and\u00a0try again.", "request_params": [{"key": "extended", "value": "1"}, {"key": "v", "value": "5.131"}, {"key": "method", "value": "groups.get"}, {"key": "oauth", "value": "1"}], "view": "alert"}}